### PR TITLE
add faq

### PR DESCRIPTION
**Pull request by prineth**

### CODE REVIEW

File: src/components/FAQ/Faq.jsx [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+  };
+
+  return (
+    <>
+      <div className="relative z-10 container px-4 mx-auto">
+        <div className="md:max-w-4xl mx-auto">
+          <HeaderTitle title={data?.headerTitle} key={data?.id} />
+          <div className="mb-11 flex flex-wrap mt-10">
+            {data?.content.map((item, i) => (
+              <div key={i} className="w-full p-1" onClick={() => toggle(i)}>
```
Comment (SanjulaD) [inline src/components/FAQ/Faq.jsx:35]: check undefined ex: content?.map

File: src/components/FAQ/Faq.jsx [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+                <div
+                  className={`py-7 px-8  ${
+                    selected === i ? "bg-brown-20 shadow-10xl" : "bg-brown-10"
+                  } `}
+                >
+                  <div className="flex flex-wrap justify-between -m-2">
+                    <div className="flex-1 p-2">
+                      <h3 className="mb-4 text-lg text-white font-semibold leading-normal">
+                        {item.question}
```
Comment (SanjulaD) [inline src/components/FAQ/Faq.jsx:44]: check undefined

File: src/components/FAQ/Faq.jsx [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+                      </h3>
+                      <p
+                        className={`text-white font-medium ${
+                          selected === i ? "block" : "hidden"
+                        }`}
+                      >
+                        {item.answer}
```
Comment (SanjulaD) [inline src/components/FAQ/Faq.jsx:51]: check undefined

File: src/shared/FaqCards/FaqCard.jsx [hunk 4]
```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import { HiMinusCircle, HiPlusCircle } from "react-icons/hi";
+
+const FaqCard = (item, i, selected) => {
+  console.log(i);
+  return (
+    <div className="flex flex-wrap justify-between -m-2">
+      <div className="flex-1 p-2">
+        <h3 className="mb-4 text-lg text-white font-semibold leading-normal">
+          {item.question}
```
Comment (SanjulaD) [inline src/shared/FaqCards/FaqCard.jsx:10]: check undefined

**Review by SanjulaD — CHANGES_REQUESTED** [review]

**Comment by SanjulaD** [comment]
image
not centered align @prineth

File: src/components/FAQ/Faq.jsx (hunk 1 above)
Comment (prineth) [reply src/components/FAQ/Faq.jsx:35]: content defined in pages/api/FaqData.js

Comment (SanjulaD) [reply src/components/FAQ/Faq.jsx:35]: as a practise when using map check undefined even the data const

**Review by SrilalS — APPROVED** [review]

**Pull request merged**